### DISCUSSION TITLE
Italian language not update in final version

### DISCUSSION THREAD

**Discussion by bovirus** [benzino77/tasmocompiler#341] [General]
@benzino77 

In italian language "it.json" the two strings

  "stepFeaturesShutterDesc": "Otturatori e tendine",
  "stepFeaturesModbusBridgeDesc": "Bridge Modbus",

appear translated in development but not in final (10.2.0) version.

As you requested I posted these changes in dev build but seems noit appleid to final version.

Thanks.

**Comment by benzino77** [comment]
That is because those changes are not merged from `development` branch to `master`.
